### ISSUE TITLE
Add a Rest API for check multiple toggles in one call

### ISSUE THREAD

**Issue by alexandrenavarro** [ff4j/ff4j#449]
**Is your feature request related to a problem? Please describe.**
Today, in our client application, we have to check 43 toggles at startup, it costs to call one by one notably because some our users are not closed to the servers, the latency is not so good (not in the same region/continent sometimes). So we want to have on call for n feature toggles.

**Describe the solution you'd like**
Add a Api in FF4jResource like

    /**
     * Check if some features are flipped
     *
     * @return
     *      Map<String,Boolean> with featureUID and flipped
     */
    @POST
    @Path("/" + OPERATION_CHECK)
    @Produces(MediaType.APPLICATION_JSON)
    @Consumes(MediaType.APPLICATION_JSON)
    @ApiOperation(value= "<b>Check</b> multiple feature toggles", response=Boolean.class)
    @ApiResponses({
            @ApiResponse(code = 200, message= "Map of feature / flipped"),
            @ApiResponse(code = 400, message= "Invalid parameter")})
    public Response check(List<String> featureUIDs) {
        // HoldSecurity Context
        FF4JSecurityContextHolder.save(securityContext);

        final Map<String, Boolean> featureFlippedMap = new HashMap<String, Boolean>();
        if (featureUIDs != null) {
            for (final String featureUID : featureUIDs) {
                boolean flipped = ff4j.check(featureUID);
                featureFlippedMap.put(featureUID, flipped);
            }
        }
        return Response.ok(featureFlippedMap).build();
    }
    

See my sample fork https://github.com/alexandrenavarro/ff4j/blob/master/ff4j-webapi/src/main/java/org/ff4j/web/api/resources/FF4jResource.java

Tell me if you are ok with the API added.

**Describe alternatives you've considered**
I'm not sure if you prefer to have a POST with requestBody with a list inside or wrapper with a list inside in case we want to add some parameter in the future (as you prefer).
We can do also a get with the list as requestParamater but I don't think it is good solution because the size of url is limited and we will have errors when the number of featureUid will be to important.

**Additional context**

PS : I also created a equivalent issue in ff4j-spring-boot-starter-parent https://github.com/ff4j/ff4j-spring-boot-starter-parent/issues/137

**Comment by alexandrenavarro** [comment]
Draft PR https://github.com/ff4j/ff4j/pull/450

**Comment by alexandrenavarro** [comment]
Thanks for the merge PR.

Nevertheless after testing on our real application a client side, we discovered a corner case I did not think about.

When you ask for multiple toggles and one is not present we sent 40x which is not really what want the client.
The use case can occur when we add a toggle feature by environment dev->uat->production and so they can be potentially different (not yet push in all environments) and the client can be not in sync with the server in all the environment notably.
You can have the feature in DEV on server side, but the client side have in DEV but also in the UAT  for instance. 
Today, the client do if the feature is unknown, it considers as disabled.

So I prefer to send a http code 200 than a 4Nx with in the map potentially (by my preference order of the solution based on what the client need)
1) non-exisiting toggle are set in the map with a value of null
2) non-exisiting toggle are set in the map with a value of false
3) only the toggles which exists are set in the map

@paul58914080 @clun 
What do you prefer on the different solutions I proposed, I will do the PR with the fix.

**Comment by paul58914080** [comment]
I prefer solution 2 which is like if not found(404) will state as false.

@clun your thoughts ?

**Comment by alexandrenavarro** [comment]
I thought about, it seems perfect for me, because it is globally what the client wants at the end.

I just created with 2 PR https://github.com/ff4j/ff4j-spring-boot-starter-parent/pull/144 and https://github.com/ff4j/ff4j/pull/454 

I also added the possibility to pass arguments (I let exactly the same API as the one with /check/$featureId), I didn't realize but I needed on my project.

Thanks in advance, don't hesitate if you have any comments.
